presto-main/src/test/java/io/prestosql/metadata/TestInformationSchemaMetadata.java

```diff
@@ -1,170 +0,0 @@
-import static io.prestosql.connector.CatalogName.createInformationSchemaCatalogName;
-import static io.prestosql.connector.CatalogName.createSystemTablesCatalogName;
-import static io.prestosql.metadata.MetadataManager.createTestMetadataManager;
-import static io.prestosql.spi.type.BigintType.BIGINT;
-import static io.prestosql.spi.type.VarcharType.VARCHAR;
-import static io.prestosql.testing.TestingSession.testSessionBuilder;
-import static io.prestosql.transaction.InMemoryTransactionManager.createTestTransactionManager;
-import static java.util.Arrays.stream;
-import static org.testng.Assert.assertEquals;
-
-public class TestInformationSchemaMetadata
-{
-    private final TransactionManager transactionManager;
-    private final Metadata metadata;
-
-    public TestInformationSchemaMetadata()
-    {
-        MockConnectorFactory.Builder builder = MockConnectorFactory.builder();
-        MockConnectorFactory mockConnectorFactory = builder.withListSchemaNames(connectorSession -> ImmutableList.of("test_schema"))
-                .withListTables((connectorSession, schemaNameOrNull) ->
-                        ImmutableList.of(
-                                new SchemaTableName("test_schema", "test_view"),
-                                new SchemaTableName("test_schema", "another_table")))
-                .withGetViews((connectorSession, prefix) -> {
-                    ConnectorViewDefinition definition = new ConnectorViewDefinition(
-                            "select 1",
-                            Optional.of("test_catalog"),
-                            Optional.of("test_schema"),
-                            ImmutableList.of(new ViewColumn("test", BIGINT.getTypeSignature())),
-                            Optional.empty(),
-                            true);
-                    SchemaTableName viewName = new SchemaTableName("test_schema", "test_view");
-                    return ImmutableMap.of(viewName, definition);
-                }).build();
-        Connector testConnector = mockConnectorFactory.create("test", ImmutableMap.of(), new TestingConnectorContext());
-        CatalogManager catalogManager = new CatalogManager();
-        String catalogName = "test_catalog";
-        CatalogName catalog = new CatalogName("test_catalog");
-        catalogManager.registerCatalog(new Catalog(
-                catalogName,
-                catalog,
-                testConnector,
-                createInformationSchemaCatalogName(catalog),
-                testConnector,
-                createSystemTablesCatalogName(catalog),
-                testConnector));
-        transactionManager = createTestTransactionManager(catalogManager);
-        metadata = createTestMetadataManager(transactionManager, new FeaturesConfig());
-    }
-
-    /**
-     * Tests information schema predicate pushdown when both schema and table name are specified.
-     */
-    @Test
-    public void testInformationSchemaPredicatePushdown()
```

Comment:
we still should have a test for actual predicate pushdown. Not only for correctness (results).

Reply:
Make sense. I reverted it back